rotkehlchen/tests/unit/decoders/test_rainbow.py

```diff
@@ -216,4 +212,302 @@ def test_rainbow_swap_token_to_token(ethereum_inquirer, ethereum_accounts):
         address=RAINBOW_ROUTER_CONTRACT,
     )]
 
-    assert expected_events == events
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0xD918F8179e915e523c0d14B98dDD45aD6AE82076']])
+def test_rainbow_swap_on_arbitrum_one(arbitrum_one_inquirer, arbitrum_one_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xeb91cb1f850ebf1ac028b9c23c2445e8d050df6eca58f9aff6a2f77e5d156fcc')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=arbitrum_one_inquirer, tx_hash=tx_hash)  # noqa: E501
+    gas_fees, swap_amount, received_amount, fee_amount, timestamp, user_address = '0.0000046705', '0.0305382', '57.839494', '0.0002618', TimestampMS(1742300433000), arbitrum_one_accounts[0]  # noqa: E501
+    assert events == [EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=0,
+        timestamp=timestamp,
+        location=Location.ARBITRUM_ONE,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_ETH,
+        amount=FVal(gas_fees),
+        location_label=user_address,
+        notes=f'Burn {gas_fees} ETH for gas',
+        counterparty=CPT_GAS,
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=1,
+        timestamp=timestamp,
+        location=Location.ARBITRUM_ONE,
+        event_type=HistoryEventType.TRADE,
+        event_subtype=HistoryEventSubType.SPEND,
+        asset=A_ETH,
+        amount=FVal(swap_amount),
+        location_label=user_address,
+        notes=f'Swap {swap_amount} ETH in Rainbow',
+        address=RAINBOW_ROUTER_CONTRACT,
+        counterparty=CPT_RAINBOW_SWAPS,
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=2,
+        timestamp=timestamp,
+        location=Location.ARBITRUM_ONE,
+        event_type=HistoryEventType.TRADE,
+        event_subtype=HistoryEventSubType.RECEIVE,
+        asset=Asset('eip155:42161/erc20:0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9'),
+        amount=FVal(received_amount),
+        location_label=user_address,
+        notes=f'Receive {received_amount} USDT as the result of a swap in Rainbow',
+        address=RAINBOW_ROUTER_CONTRACT,
+        counterparty=CPT_RAINBOW_SWAPS,
+    ), EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=3,
+        timestamp=timestamp,
+        location=Location.ARBITRUM_ONE,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_ETH,
+        amount=FVal(fee_amount),
+        location_label=user_address,
+        notes=f'Spend {fee_amount} ETH a Rainbow fee',
```

Comment:
This has a typo. Fix at the decoder source

```suggestion
        notes=f'Spend {fee_amount} ETH as Rainbow fee',
```